app/views/shared/_movements.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="panel user-profile">
+<div class="panel user-profile panel-default">
```

Comment:
Again, looks like you're mixing several patterns into one. Using BEM, an elegant way to solve this would be:

```
<div class="panel panel--user-profile panel--default">
```